modules/i18n/src/translations/en-US/portals/user.ts

```diff
@@ -568,6 +579,30 @@ export const user: userNS = {
                     message: "Unable to trigger a force password reset"
                 }
             },
+            resendCode: {
+                genericError: {
+                    description: "Error occurred while resending the recovery code.",
```

Comment:
can we make sure that a recovery code is sent always? wouldn't the admins be able to configure to send an email link instead? if they would, we may need to dynamically show a suitable message depending on the recovery method.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/399e7573e645d5c76ea0e9c6b3a47780a2c40019